packages/spark-core/components/_buttons.scss

```diff
@@ -109,3 +109,21 @@
     color: $btn-disabled-text-color;
   }
 }
+
+.sprk-c-Button--secondary.sprk-is-Disabled,
+.sprk-c-Button--secondary[disabled] {
+  box-shadow: none;
+
+  background-color: $btn-secondary-disabled-background-color;
+  border-color: $btn-secondary-disabled-border-color;
+  color: $btn-secondary-disabled-text-color;
+}
+
+.sprk-c-Button--tertiary.sprk-is-Disabled,
+.sprk-c-Button--tertiary[disabled] {
+  box-shadow: none;
```

Comment:
These box-shadows and borders need to be set as vars and zeroed out in the settings file.